libraries/apollo-gradle-plugin/src/test/kotlin/util/TestUtils.kt

```diff
@@ -176,8 +176,15 @@ object TestUtils {
         .forwardStdOutput(output)
         .forwardStdError(error)
         .withProjectDir(projectDir)
+        // Runs in-process, saves a bit of time
         .withDebug(true)
-        .withArguments("--stacktrace", *args)
+        .withArguments(
+            "-g",
+            // Reuse the main build home directory
```

Comment:
I would recommend using a copy of this directory. I am afraid that sharing this directory with the test process could lead to a deadlock.

Reply:
The thing is local machines. `~/.gradle/caches/modules-2` is pretty large on local machines. On my machine, it's currently 15GB and I don't really want to copy that every time I run the Gradle integration tests. 
I thought several Gradle runs could share the same Gradle home?